client/scripts/helpers/calculator_helper.ts

```diff
@@ -0,0 +1,144 @@
+import moment from 'moment-twitter'
+import {AssetInfo} from '../models/AssetInfo'
+
+// duplicated in helpers.ts
+// changes:  num.tofixed(2) 
+export function formatNumberShort(num: number) {
+    const round = (n, digits?) => {
+        if (digits === undefined) digits = 2
+        return Math.round(n * Math.pow(10, digits)) / Math.pow(10, digits)
+    }
+
+    const precise = (n, digits?) => {
+        if (digits === undefined) digits = 3
+        return n.toPrecision(digits)
+    }
+
+    return num > 1_000_000_000_000 ? round(num / 1_000_000_000_000).toFixed(2) + 't'
+        : num > 1_000_000_000 ? round(num / 1_000_000_000).toFixed(2) + 'b'
+            : num > 1_000_000 ? round(num / 1_000_000).toFixed(2) + 'm'
+                : num > 1_000 ? round(num / 1_000).toFixed(2) + 'k'
+                    : num > 0.1 ? round(num).toFixed(2)
+                        : num > 0.01 ? Number(precise(num, 2)).toFixed(2)
+                            : num > 0.001 ? Number(precise(num, 1)).toFixed(2)
+                                : num.toFixed(2)
+}
+
+// duplicated in client/scripts/helpers/index.ts
```

Comment:
same here?

Reply:
formatDuration was missing Year, Month and Days.
Also i was not adding seconds in UI

Reply:
ok

Reply:
no way to merge them?

Reply:
Yes, I can merge them, but that will need us to validate all pages where it is used.